### PR TITLE
Update pdf building

### LINKED ISSUE #53: Update PDF Building
**Problem:** Current review pages and PDF are not up to client specs
**Solution:** Fix the review pages and PDF building function
**Description:**
Currently, our review page contains all of their answers. This page should only contain the legal text for complaints they have identified, and the additional details provided in the prompts section. The same should be true for the PDF built at the end of the form.
**Specs:**
- [x] Review page
  - [x] Should only include questions of type TEXT
  - [x] Should only include the portion of those questions with the legal text included. This can be filtered in one of two ways:
    - We can update the flowchart and flowchart parsing system to separate out the legal text from the rest - this is the more difficult approach
    - We can simply parse the question and look for quotation marks - there should be quotation marks around the legal text for each of these questions - this is the easier approach
- [x] PDF
  - [x] Should only include questions of type TEXT
  - [x] Should only include the portion of those questions with the legal text included
  - [x] Make the name better, such as PRS Complaint.pdf
  - [x]  Make sure that the formatting of PDF is good

**Resources:** [jsPDF documentation](http://raw.githack.com/MrRio/jsPDF/master/docs/index.html), [Figma mockup of what the ending PDF would look like](https://www.figma.com/file/sz9vT4i5QbYOSxLfVzCNXJ/Desktop-Mockups?node-id=242%3A10)
**If blocked:** @dillydally414 or #engineering-support

### PR DESCRIPTION
**Pull request by clue4**
closes #53 issue

### CODE REVIEW

**Comment by clue4** [comment]
pdf building def a bit jank and prob not very robust ;-;

**Review by dillydally414 — APPROVED** [review]
Looks great!

File: pages/DynamicPOC/index.tsx [hunk 1]
```diff
@@ -160,16 +160,22 @@ const DynamicPOC: React.FC<{ questions: Question[] }> = ({ questions }) => {
...
+      }
       if (item.answer != null) {
         doc.setFont('times', 'normal').text('\t' + item.answer + '\n\n', x, y)
         y += y_inc
       }
       if (item.formAnswer.type === QuestionType.TEXT) {
-        doc
-          .setFont('times', 'normal')
-          .text('\t' + item.formAnswer.userAnswer + '\n\n', x, y)
+        const splitAnswer = doc.splitTextToSize(item.formAnswer.userAnswer, 200)
```
Comment (devashish9) [inline pages/DynamicPOC/index.tsx:173]: Is this a function to help write the formanswers into a pdf?

**Review by devashish9 — COMMENTED** [review]
Everything looks good, just curious about what the splitanswer functions do

File: pages/DynamicPOC/index.tsx (hunk 1 above)
Comment (clue4) [reply pages/DynamicPOC/index.tsx:173]: yup, it's a method in jspdf to split the text so it doesn't go off the page

**Review by imanmoreira — APPROVED** [review]
love it

File: components/DynamicForm/MyResult.tsx [hunk 2]
```diff
@@ -27,15 +27,17 @@ export function buildResults(
...
+      )
         return results
 
       const contentBasedFormAnswer: FormResult = {
-        answer:
-          curFormAns.type == QuestionType.RADIO
-            ? curQuestion.answers[curFormAns.answerId].content
-            : undefined,
-        question: curQuestion.question,
+        answer: undefined,
```
[the remark this answers is not in the document]
Comment (clue4) [reply components/DynamicForm/MyResult.tsx:39]: only needed that for radios,which we're not showing anymore

**Pull request merged**